src/client/client4.js

```diff
@@ -1584,6 +1584,13 @@ export default class Client4 {
         );
     };
 
+    editOAuthApp = async (app) => {
+        return this.doFetch(
+            `${this.getOAuthAppsRoute()}/${app.id}`,
```

Comment:
looks like this route does not exists? thus the build failure

```
1) Actions.Integrations editOAuthApp:
     Error: {"message":"Sorry, we could not find the page.","server_error_id":"api.context.404.app_error","status_code":404,"url":"http://localhost:40914/api/v4/oauth/apps/qtodcceyfj8hden61g6ifiq6jc"}
      at _callee22$ (test/actions/integrations.test.js:499:19)
      at tryCatch (node_modules/babel-polyfill/node_modules/regenerator-runtime/runtime.js:65:40)
      at Generator.invoke [as _invoke] (node_modules/babel-polyfill/node_modules/regenerator-runtime/runtime.js:303:22)
      at Generator.prototype.(anonymous function) [as next] (node_modules/babel-polyfill/node_modules/regenerator-runtime/runtime.js:117:21)
      at step (test/actions/integrations.test.js:33:191)
      at test/actions/integrations.test.js:33:361
      at process._tickCallback (internal/process/next_tick.js:103:7)
```

Reply:
It does not exist yet but I have PR for that https://github.com/mattermost/platform/pull/7361
I'll rerun build once it gets merged